tests/kernel/timer/timer_api/testcase.yaml

```diff
@@ -4,5 +4,5 @@ tests:
   - test_tickless:
       build_only: true
       extra_args: CONF_FILE="prj_tickless.conf"
-      filter: CONFIG_BOARD_QEMU_X86
+      filter: (CONFIG_BOARD_QEMU_X86 or CONFIG_XTENSA)
```

Comment:
Shouldn't this be CONFIG_BOARD_QEMU_XTENSA instead?

Reply:
To run on ESP32 "CONFIG_XTENSA" is required. Or should we make the application to run only on QEMU, to make it generic?